packages/components/src/Components/InsightsLabel/__snapshots__/InsightsLabel.test.js.snap

```diff
@@ -0,0 +1,21 @@
+// Jest Snapshot v1, https://goo.gl/fbAQLP
+
+exports[`InsightsLabel component should render defaults 1`] = `
+<Component
+  className="ins-c-label-1"
+  icon={false}
+/>
+`;
+
+exports[`InsightsLabel component should render with icon 1`] = `
+<Component
```

Comment:
You should change `shallow` to `mount` or use `shallow.dive()`. These snapshots are not very useful.